ui/apps/platform/src/Containers/VulnMgmt/List/Cves/CveBulkActionDialogue.js

```diff
@@ -46,16 +52,61 @@ const CveBulkActionDialogue = ({ closeAction, bulkActionCveIds }) => {
     const [policies, setPolicies] = useState([]);
 
     // use GraphQL to get the (hopefully cached) cve summaries to display in the dialog
-    const CVES_QUERY = gql`
-        query getCves($query: String) {
-            results: vulnerabilities(query: $query) {
-                id
-                cve
-                summary
-                vulnerabilityTypes
-            }
+    let cveQuery = '';
```

Comment:
can we stick to the all caps convention as we have before for the gqls for easy parsing?